config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  resources :uploaded_files
```

Comment:
This creates all CRUD routes. Are they all needed at this point? Also, since the attachments should be related to a specific MembershipApplication, I would suggest making these routes as part or that resource (by nesting them).